app/src/androidTest/java/org/mozilla/focus/activity/OpenwithDialogTest.java

```diff
@@ -73,7 +73,12 @@ public void OpenTest() throws InterruptedException, UiObjectNotFoundException {
         TestHelper.inlineAutocompleteEditText.setText("mozilla");
         TestHelper.hint.waitForExists(waitingTime);
         TestHelper.pressEnterKey();
-        assertTrue(TestHelper.webView.waitForExists(waitingTime));
+        if (TestHelper.getAppName().contains("webview")) {
+            assertTrue(TestHelper.webView.waitForExists(waitingTime));
+        }
```

Comment:
\[Checkstyle\] ERROR: '\}' at column 9 should be on the same line as the next part of a multi\-block statement\.